// angular-text-input-highlight/__ivy_ngcc__/text-input-highlight.module.js
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { TextInputHighlightComponent } from './text-input-highlight.component';
import { TextInputHighlightContainerDirective } from './text-input-highlight-container.directive';
import { TextInputElementDirective } from './text-input-element.directive';
import * as ɵngcc0 from '@angular/core';
var TextInputHighlightModule = /** @class */ (function () {
    function TextInputHighlightModule() {
    }
    /** @nocollapse */
    TextInputHighlightModule.ctorParameters = function () { return []; };
TextInputHighlightModule.ɵfac = function TextInputHighlightModule_Factory(t) { return new (t || TextInputHighlightModule)(); };
TextInputHighlightModule.ɵmod = /*@__PURE__*/ ɵngcc0.ɵɵdefineNgModule({ type: TextInputHighlightModule });
TextInputHighlightModule.ɵinj = /*@__PURE__*/ ɵngcc0.ɵɵdefineInjector({ imports: [[CommonModule]] });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(TextInputHighlightModule, [{
        type: NgModule,
        args: [{
                declarations: [
                    TextInputHighlightComponent,
                    TextInputHighlightContainerDirective,
                    TextInputElementDirective
                ],
                imports: [CommonModule],
                exports: [
                    TextInputHighlightComponent,
                    TextInputHighlightContainerDirective,
                    TextInputElementDirective
                ]
            }]
    }], function () { return []; }, null); })();
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(TextInputHighlightModule, { declarations: function () { return [TextInputHighlightComponent,
        TextInputHighlightContainerDirective,
        TextInputElementDirective]; }, imports: function () { return [CommonModule]; }, exports: function () { return [TextInputHighlightComponent,
        TextInputHighlightContainerDirective,
        TextInputElementDirective]; } }); })();
    return TextInputHighlightModule;
}());
export { TextInputHighlightModule };
