// angular-text-input-highlight/__ivy_ngcc__/text-input-highlight-container.directive.js
import { Directive } from '@angular/core';
import * as ɵngcc0 from '@angular/core';
var TextInputHighlightContainerDirective = /** @class */ (function () {
    function TextInputHighlightContainerDirective() {
    }
    /** @nocollapse */
    TextInputHighlightContainerDirective.ctorParameters = function () { return []; };
TextInputHighlightContainerDirective.ɵfac = function TextInputHighlightContainerDirective_Factory(t) { return new (t || TextInputHighlightContainerDirective)(); };
TextInputHighlightContainerDirective.ɵdir = /*@__PURE__*/ ɵngcc0.ɵɵdefineDirective({ type: TextInputHighlightContainerDirective, selectors: [["", "mwlTextInputHighlightContainer", ""]], hostVars: 2, hostBindings: function TextInputHighlightContainerDirective_HostBindings(rf, ctx) { if (rf & 2) {
        ɵngcc0.ɵɵclassProp("text-input-highlight-container", true);
    } } });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(TextInputHighlightContainerDirective, [{
        type: Directive,
        args: [{
                selector: '[mwlTextInputHighlightContainer]',
                host: {
                    '[class.text-input-highlight-container]': 'true'
                }
            }]
    }], function () { return []; }, null); })();
    return TextInputHighlightContainerDirective;
}());
export { TextInputHighlightContainerDirective };
