// angular-text-input-highlight/__ivy_ngcc__/text-input-element.directive.js
import { Directive } from '@angular/core';
import * as ɵngcc0 from '@angular/core';
var TextInputElementDirective = /** @class */ (function () {
    function TextInputElementDirective() {
    }
    /** @nocollapse */
    TextInputElementDirective.ctorParameters = function () { return []; };
TextInputElementDirective.ɵfac = function TextInputElementDirective_Factory(t) { return new (t || TextInputElementDirective)(); };
TextInputElementDirective.ɵdir = /*@__PURE__*/ ɵngcc0.ɵɵdefineDirective({ type: TextInputElementDirective, selectors: [["textarea", "mwlTextInputElement", ""]], hostVars: 2, hostBindings: function TextInputElementDirective_HostBindings(rf, ctx) { if (rf & 2) {
        ɵngcc0.ɵɵclassProp("text-input-element", true);
    } } });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(TextInputElementDirective, [{
        type: Directive,
        args: [{
                selector: 'textarea[mwlTextInputElement]',
                host: {
                    '[class.text-input-element]': 'true'
                }
            }]
    }], function () { return []; }, null); })();
    return TextInputElementDirective;
}());
export { TextInputElementDirective };
